### ISSUE TITLE
Unable to use @twittwer/compodoc in a new nx workspace

### ISSUE THREAD

**Issue by bskeen** [twittwer/nx-tools#37]
I am getting an error when trying to use @twittwer/compodoc with a new nx workspace. These are the steps I follow:

1. `npx create-nx-workspace@latest`
  - I provide the workspace name, select Angular for the project type to generate, and set the other options.
2. `cd <workspace>`
3. `nx add @twittwer/compodoc`
4. `nx g @twittwer/compodoc:config <project-name> --workspaceDocs`
5. `nx run <project-name>:compodoc:json`

I get this warning during step 3:

```
"compodoc-add" schema is using the keyword "id" which its support is deprecated. Use "$id" for schema ID.
```

The rest of the commands seem to work fine, but the last command errors out with:

```
schema with key or id "http://json-schema.org/draft-07/schema" already exists
```

Am I missing a step? Or is there a bug? I am new to both nx and storybook (it's my goal to get that working by using @twittwer/compodoc), so it could be that I misunderstand some setup that is needed to get this working.

**Comment by twittwer** [comment]
Thx for the info, seems like the id property has to be renamed - I will try to reproduce it tomorrow. Which NX version and OS are you working on?

**Comment by bskeen** [comment]
Thanks for getting back to me. I am using the latest NX (looks like it is 12.3.3). I tried it on Windows 10, OS X Big Sur, and Ubuntu (on WSL 2), and they all had the same issue.

**Comment by twittwer** [comment]
The id problem is fixed in 1.5.2-beta.1
I'm checking that it works correctly with the solutions tsconfigs in apps (1.5.2-beta.2).
I will let you know when a non-beta release is ready.

**Comment by twittwer** [comment]
1.5.2 is released 🎉

I tested it on OSX & Windows.
